lib/tapioca/compilers/requires_compiler.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 # typed: strict
 
-require_relative '../sorbet_config_parser'
+require 'spoom'
```

Comment:
You probably don't need this since `Spoom` is loaded by default.

Reply:
Not require it gives me a "uninitialized constant `::Spoom`", should it not do it? Maybe something is wrong with my gem, I can investigate if it is supposed to be working.

Reply:
It seems you **do** need it after all. My bad, sorry for the confusion.